config/configloader/testdata/empty-all-sections.yaml

```diff
@@ -2,4 +2,4 @@ receivers:
 exporters:
 processors:
 service:
-  pipeline:
+  pipelines:
```

Comment:
Can you send this fix as a separate PR?

Reply:
https://github.com/open-telemetry/opentelemetry-collector/pull/3359